sale_procurement_group_by_line/model/sale.py

```diff
@@ -49,7 +49,15 @@ def action_ship_create(self, cr, uid, ids, context=None):
             groups = {}
 
             for line in order.order_line:
-                group_id = groups.get(line._get_procurement_group_key())
+                if order.state == 'shipping_except':
+                    try:
```

Comment:
You are right @pedrobaeza . This can be removed. I'll make the change and also need to check why CI is failing.